### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #1048 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#465]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 7.87 KB...]
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.076 sec - in smartcity.accessibility.socialnetwork.UserFunctionalityTest
Running smartcity.accessibility.socialnetwork.BestReviewsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.01 sec - in smartcity.accessibility.socialnetwork.BestReviewsTest
Running smartcity.accessibility.socialnetwork.UserPinTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.076 sec - in smartcity.accessibility.socialnetwork.UserPinTest
Running smartcity.accessibility.socialnetwork.DeleteReviewsTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.125 sec - in smartcity.accessibility.socialnetwork.DeleteReviewsTest
Running smartcity.accessibility.socialnetwork.ScoreTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.768 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.659 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.872 sec <<< FAILURE! - in smartcity.accessibility.database.LocationManagerTest
testGetNonAccessibleLocationsInRadius(smartcity.accessibility.database.LocationManagerTest)  Time elapsed: 0.094 sec  <<< FAILURE!
java.lang.AssertionError: expected:<2> but was:<0>
	at smartcity.accessibility.database.LocationManagerTest.testGetNonAccessibleLocationsInRadius(LocationManagerTest.java:169)

Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.367 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.327 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.602 sec - in smartcity.accessibility.search.SearchTest

Results :

Failed tests: 
  LocationManagerTest.testGetNonAccessibleLocationsInRadius:169 expected:<2> but was:<0>

Tests run: 88, Failures: 1, Errors: 0, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:23 min
[INFO] Finished at: 2017-06-30T13:11:07+00:00
[INFO] Final Memory: 34M/199M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] a9eb53824bdc366dfa78c614d2abe6c5ee50b9fc - fix bug in get non acceissable location in DB

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/1048/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
